website/content/en/preview/concepts/setup.md

```diff
@@ -0,0 +1,554 @@
+                  "aws:ResourceTag/karpenter.sh/provisioner-name": "*"
+                }
+              }
+            },
+```
+
+### AllowRegionalReadActions
+
+The AllowRegionalReadActions Sid allows [DescribeAvailabilityZones](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeAvailabilityZones.html), [DescribeImages](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeImages.html), [DescribeInstances](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html), [DescribeInstanceTypeOfferings](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstanceTypeOfferings.html), [DescribeInstanceTypes](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstanceTypes.html), [DescribeLaunchTemplates](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeLaunchTemplates.html), [DescribeSecurityGroups](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeSecurityGroups.html), [DescribeSpotPriceHistory](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeSpotPriceHistory.html), and [DescribeSubnets](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeSubnets.html) actions for the current AWS region.
+This allows the Karpenter controller to do any of those read-only actions across all related resources for that AWS region.
+
+```
+            {
+              "Sid": "AllowRegionalReadActions",
+              "Effect": "Allow",
+              "Resource": "*",
+              "Action": [
+                "ec2:DescribeAvailabilityZones",
+                "ec2:DescribeImages",
+                "ec2:DescribeInstances",
+                "ec2:DescribeInstanceTypeOfferings",
+                "ec2:DescribeInstanceTypes",
+                "ec2:DescribeLaunchTemplates",
+                "ec2:DescribeSecurityGroups",
+                "ec2:DescribeSpotPriceHistory",
+                "ec2:DescribeSubnets"
+              ],
+              "Condition": {
+                "StringEquals": {
+                  "aws:RequestedRegion": "${AWS::Region}"
+                }
+              }
+            },
+```
+
+### AllowGlobalReadActions
+Because pricing information does not exist in every region at the moment, the AllowGlobalReadActions Sid allows the Karpenter controller to get product pricing information (`pricing:GetProducts`) for all related resources across all regions.
+The `ssm:GetParameter" action lets the Karpenter controller read SSM parameters.
+
+**NOTE**: If potentially sensitive information is stored in SSM parameters, you could consider restricting access to these messages further.
+
+```
+            {
+              "Sid": "AllowGlobalReadActions",
+              "Effect": "Allow",
+              "Resource": "*",
+              "Action": [
+                "pricing:GetProducts",
+                "ssm:GetParameter"
+              ]
+            },
+```
+
+### AllowInterruptionQueueActions
```

Comment:
There's been no mention of an interruption queue that's been created yet, so I find this statement a little confusing. Perhaps this statement should talk about the fact that Karpenter supports interruption queues (as detailed in the interruption section of the deprovisioning conceptual docs) and that you can give Karpenter permission to access that queue by specifying the resource ARN in the CF template.

Reply:
I took a stab at rewording this.